packages/core/src/components/forms/controls.tsx

```diff
@@ -62,6 +79,7 @@ export class Control<P extends IControlProps> extends React.Component<React.HTML
                 <input {...inputProps} ref={inputRef} type={type} />
                 <span className={Classes.CONTROL_INDICATOR} />
                 {this.props.label}
+                {this.props.labelElement}
```

Comment:
why? it's just JSX children, there's no limit. seems more error-prone to only render one.